content/english/security/2-password-basics/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: "Password Basics"
+description: ""
+draft: true
+weight: 2
+translationKey: "security-2"
+---
+
+Our Secret Agent Alice solves the case by explaining Bob about Passowrds!!!
+
+## What is a Password?
+
+Passwords are the key to the lock to your virtual world.
+Just like your secret is kept safe in a secure room using a lock and a key similarly your personal information on the internet is secured by your login password and username. 
+The secure room is only as secure as the key to the lock on the door. We need stronger passwords to protect our online data and identity from bad people known as **Hackers**.
+
+
+
+## What went wrong?
+
+Passwords need to be private and should only be known to you. A strong password is one that is more secure by virtue of being difficult for a machine or a human to guess. 
+
+#### Examples of weak passwords
+1. Any common dictionary word, in any language (e.g., airplane or aeroplano).
```

Comment:
I would add "Password" as an example of a weak password :)

Reply:
added it!